src/adhocracy_core/adhocracy_core/sheets/__init__.py

```diff
@@ -215,8 +215,15 @@ def get_cstruct(self, request: Request, params: dict={}):
         Automatically set params are: `only_visible` and `allows` view
         permission.
         """
-        params['allows'] = (request.effective_principals, 'view')
-        params['only_visible'] = True
+        params = params or {}
+        filter_view_permission = self.registry.settings.get(
```

Comment:
This needs to go through `asbool`, otherwise `"False"` would evaluate to `True`. Same further down.